x-pack/plugins/security_solution/common/detection_engine/schemas/request/perform_bulk_action_schema.ts

```diff
@@ -6,13 +6,28 @@
  */
 
 import * as t from 'io-ts';
-import { bulkAction, queryOrUndefined } from '../common/schemas';
+import { BulkAction, queryOrUndefined, bulkActionUpdate } from '../common/schemas';
 
-export const performBulkActionSchema = t.exact(
-  t.type({
-    query: queryOrUndefined,
-    action: bulkAction,
-  })
-);
+export const performBulkActionSchema = t.union([
+  t.exact(
+    t.type({
+      query: queryOrUndefined,
+      action: t.union([
+        t.literal(BulkAction.delete),
+        t.literal(BulkAction.disable),
+        t.literal(BulkAction.duplicate),
+        t.literal(BulkAction.enable),
+        t.literal(BulkAction.export),
+      ]),
+    })
+  ),
+  t.exact(
+    t.type({
+      query: queryOrUndefined,
```

Comment:
We will need to support applying bulk edits by a list of rule IDs, just double-checking is the plan to add this in a follow-up PR?

Reply:
in follow-up PRs, when will be building UI, so ids could be passed when user select rules on a page